README.md

```diff
@@ -104,6 +104,10 @@ Hertz is distributed under the [Apache License, version 2.0](https://github.com/
 - WeChat: CloudWeGo community WeChat group.
 
 ![WechatGroup](images/wechat_group_cn.png)
+## Contributors
+[![Contributors](https://contrib.rocks/image?repo=cloudwego/hertz)](https://github.com/cloudwego/hertz)
+## Stargazers
```

Comment:
I don't think Stargazers are needed at this stage

Reply:
> I don't think Stargazers are needed at this stage

done